components/camel-soap/pom.xml

```diff
@@ -37,6 +37,8 @@
             org.apache.camel*;version=${camel.version}
         </camel.osgi.export>
         <camel.osgi.import>
+            !org.w3._2003._05.soap_envelope,
+            !org.xmlsoap.schemas.soap.envelope,
```

Comment:
Added this exclusions because these packages are included in the component and are required

Reply:
This is not the right approach indeed if you look at the content of the artifact `camel-soap`, you will see that it contains those packages so the right approach is to export them

Reply:
understood, works this way, fixed, thanks!